indexer/services/vulcan/src/handlers/order-place-handler.ts

```diff
@@ -157,20 +157,6 @@ export class OrderPlaceHandler extends Handler {
       };
       sendMessageWrapper(subaccountMessage, KafkaTopics.TO_WEBSOCKETS_SUBACCOUNTS);
     }
-
-    // TODO(IND-68): Remove once order replacement flow in V4 protocol removes the old order and
-    // places the updated order.
-    if (updatedQuantums !== undefined) {
-      const orderbookMessage: Message = {
-        value: this.createOrderbookWebsocketMessage(
-          placeOrderResult.oldOrder!,
-          perpetualMarket,
-          updatedQuantums,
-        ),
-        headers,
-      };
-      sendMessageWrapper(orderbookMessage, KafkaTopics.TO_WEBSOCKETS_ORDERBOOKS);
-    }
```

Comment:
Removing because this is sending a websocket message to remove the unfilled amount of the original order from the orderbook price level. 

On order replacement there are 2 cases:
1. If the price is different, the original order will get removed through the order removal handler, so we don't need this here
2. When the price is the same, we shouldn't remove the unfilled amount of the original order from the orderbook and then re-add the replacement order size to the orderbook because this is 2 websocket messages and can cause flickering. We should just wait for the update message we expect after this order place to update the orderbook price-level size

Reply:
Another option is to not update the OrderbookLevelsCache as part of this order place handler when the order is replaced and just rely on the order update message to handle it